autoload/gitgutter.vim

```diff
@@ -38,6 +38,10 @@ function! gitgutter#process_buffer(bufnr, force) abort
 
   if gitgutter#utility#is_active(a:bufnr)
     if a:force || s:has_fresh_changes(a:bufnr)
+      if a:force
+        let p = gitgutter#utility#repo_path(a:bufnr, 0)
```

Comment:
Why is this line necessary?

Reply:
See the test case above..

I might have mixed up something here though with another PR (e.g. the return value), not sure currently.